src/pages/patientView/clinicalInformation/lib/clinicalAttributesStyleConsts.js

```diff
@@ -0,0 +1,285 @@
+/**
+ * Enums and configuration object mapping.
+ */
+// Colors taken from clinicalAttributes.scss
+export const colors = {
+    sample_color_primary: 'black',
+    sample_color_recurrence: 'orange',
+    sample_color_metastasis: 'red',
+    sample_color_cfdna: 'blue',
+    sample_color_xenograft: 'pink',
+};
+
+/**
+ * Object mapping of keys to objects indicating
+ * whether an attribute is primary or secondary,
+ * the formatter to display the inputted value t,
+ * and font coloring based on the key and value.
+ */
+export const config = {
+    AGE: {
+        primary: true,
+        formatter: t => {
+            return `${t} years old`;
+        },
+        order: 2,
+    },
+    BREAST_CANCER_RECEPTOR_STATUS: {
+        order: 6,
+    },
+    CANCER_TYPE: {
+        primary: true,
+        order: 3,
+    },
+    CANCER_TYPE_DETAILED: {
+        primary: false,
+        order: 4,
+    },
+    DERIVED_NORMALIZED_CASE_TYPE: {
+        primary: true,
+        values: {
+            Primary: colors.sample_color_primary,
+            Progressed: colors.sample_color_recurrence,
+            Recurrence: colors.sample_color_recurrence,
+            Metastasis: colors.sample_color_metastasis,
+        },
+        order: 1,
+    },
+    DERIVED_SAMPLE_LOCATION: {
+        primary: false,
+        order: 2,
+    },
+    DFS_STATUS: {
+        primary: true,
+        values: {
+            _: '#f00',
+            DiseaseFree: 'rgb(0, 128, 0)',
+            Yes: 'rgb(0, 128, 0)',
+        },
```

Comment:
This is really minor and very debatable, but if you are always iterating through all the attributes in `values`, I'd rather it be an array of key value pairs. I think an array of key value pairs shows your intent a bit better.
```suggestion
        values: [
           {key: '_', value: '#f00'},
           {key: 'DiseaseFree', value: 'rgb(0, 128, 0)'}
        },
```

Reply:
i think the map is fine @Luke-Sikina, it's a lookup table.  we do that a lot